src/Avatar/index.tsx

```diff
@@ -1,20 +1,25 @@
 import { Avatar as AntdAvatar, AvatarProps as AntdAvatarProps } from 'antd';
 import React from 'react';
 
+import { Tooltip } from '../Tooltip';
 import { MLL_THEME } from '../theme';
 
 import { stringToHslaColor } from './utils';
 
 export const Avatar: typeof AntdAvatar = AntdAvatar;
 export type AvatarProps = AntdAvatarProps;
 
-export function UserAvatar(props: AvatarProps & { username: string }) {
-  const color = stringToHslaColor(props.username);
+type AvatarUserProps = { toolTipLabel?: string; initials: string };
+
+export function UserAvatar(props: AvatarProps & AvatarUserProps) {
+  const color = stringToHslaColor(props.initials);
   return (
-    <Avatar
-      style={{ backgroundColor: color, color: MLL_THEME.backgroundColor }}
-    >
-      {props.username}
-    </Avatar>
+    <Tooltip title={props.toolTipLabel ?? null}>
+      <Avatar
+        style={{ backgroundColor: color, color: MLL_THEME.backgroundColor }}
+      >
+        {props.initials}
```

Comment:
As this is the primarily displayed content, I would expect this:

```suggestion
        {props.children}
```